Cargo.toml

```diff
@@ -1,12 +1,88 @@
-[package]
-name = "argent-contracts-starknet"
-version = "0.1.0"
-edition = "2021"
+[workspace]
+
+members = [
+    "cairo/crates/cairo-lang-casm",
+    "cairo/crates/cairo-lang-compiler",
+    "cairo/crates/cairo-lang-utils",
+    "cairo/crates/cairo-lang-debug",
+    "cairo/crates/cairo-lang-defs",
+    "cairo/crates/cairo-lang-proc-macros",
+    "cairo/crates/cairo-lang-diagnostics",
+    "cairo/crates/cairo-lang-eq-solver",
+    "cairo/crates/cairo-lang-filesystem",
+    "cairo/crates/cairo-lang-formatter",
+    "cairo/crates/cairo-lang-language-server",
+    "cairo/crates/cairo-lang-lowering",
+    "cairo/crates/cairo-lang-parser",
+    "cairo/crates/cairo-lang-plugins",
+    "cairo/crates/cairo-lang-project",
+    "cairo/crates/cairo-lang-runner",
+    "cairo/crates/cairo-lang-semantic",
+    "cairo/crates/cairo-lang-sierra-ap-change",
+    "cairo/crates/cairo-lang-sierra-gas",
+    "cairo/crates/cairo-lang-sierra-generator",
+    "cairo/crates/cairo-lang-sierra-to-casm",
+    "cairo/crates/cairo-lang-sierra",
+    "cairo/crates/cairo-lang-starknet",
+    "cairo/crates/cairo-lang-syntax-codegen",
+    "cairo/crates/cairo-lang-syntax",
+    "cairo/crates/cairo-lang-test-runner",
+]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[dependencies]
+[workspace.package]
+version = "1.0.0-alpha.2"
+edition = "2021"
+repository = "https://github.com/starkware-libs/cairo/"
+license = "Apache-2.0"
+license-file = "LICENSE"
 
-[[bin]]
-name="contracts"
-path = "contracts/lib.cairo"
\ No newline at end of file
+[workspace.dependencies]
+anyhow = "1.0.66"
+ark-ff = "0.4.0-alpha.7"
+ark-std = "0.3.0"
+assert_matches = "1.5"
+bimap = "0.6.2"
+cairo-felt = "0.1.1"
+cairo-vm = "0.1.2"
+chrono = "0.4.23"
+clap = { version = "4.0", features = ["derive"] }
+colored = "2"
+const-fnv1a-hash = "1.1.0"
+convert_case = "0.6.0"
+derivative = "2.2.0"
+diffy = "0.3.0"
+env_logger = "0.9.3"
+genco = "0.17.0"
+good_lp = { version = "1.3.2", features = ["minilp"], default-features = false }
+id-arena = "2.2.1"
+indexmap = "1.9.1"
+indoc = "1.0.7"
+itertools = "0.10.3"
+lalrpop-util = { version = "0.19.8", features = ["lexer"] }
+log = "0.4"
+lsp = { version = "0.93", package = "lsp-types" }
+num-bigint = "0.4"
+num-traits = "0.2"
+num-integer = "0.1"
+path-clean = "0.1.0"
+pretty_assertions = "1.2.1"
+proc-macro2 = "1.0"
+quote = "1.0.21"
+rayon = "0.9.0"
+salsa = "0.16.1"
+serde = { version = "1.0.130", features = ["derive"] }
+serde_json = "1.0"
+sha3 = "0.10.6"
+smol_str = "0.1.23"
+substring = "1.4.5"
+syn = { version = "1.0.99", features = ["full", "extra-traits"] }
+test-case = "2.2.2"
+test-case-macros = "2.2.2"
+test-log = "0.2.11"
+thiserror = "1.0.32"
+tokio = { version = "1.18.2", features = ["full", "sync"] }
+toml = "0.4.2"
+tower-lsp = "0.17.0"
+unescaper = "0.1.1"
+xshell = "0.2.2"
```

Comment:
are these dependecies copied from the cairo repo? will this need to be updated manually?

Reply:
Since the use some kind of virtual toml file, atm it is a copy paste of theirs.
I guess once they have a stable version it should be as easy as putting
starknet-lang = "1.0" 
int the fiel